sdk/translation/azure-ai-translation-document/CHANGELOG.md

```diff
@@ -1,12 +1,15 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 2.0.0-beta.1 (2024-05-07)
 
 ### Features Added
+- `documenttranslate` is the method added to submit a single/synchronous document translation request to the Document Translation service.
+- `DocumentTranslateContent` has been added to specify the `documenttranslate` request content.
+- Added `getSupportedFormats` that returns a list of supported documents and glossaries by the Document Translation service.
```

Comment:
Please double check these entries are accurate for the python library since they look copy/pasted from the C# SDK.

Reply:
Same for the entries below

Reply:
Updated